Makefile

```diff
@@ -44,33 +49,60 @@ test:
 functional-test:
 	go test -mod=vendor -timeout=120s -v -tags functional -cover ./local-container-endpoints/handlers/functional_tests/...
 
-$(LINUX_BINARY): $(SOURCES)
+$(AMD_BINARY): $(SOURCES)
 	@mkdir -p ./bin/linux-amd64
 	TARGET_GOOS=linux GOARCH=amd64 ./scripts/build_binary.sh ./bin/linux-amd64
-	@echo "Built local-container-endpoints for linux"
+	@echo "Built local-container-endpoints for linux-amd64"
+
+$(AMD_BINARY): $(SOURCES)
+	@mkdir -p ./bin/linux-arm64
+	TARGET_GOOS=linux GOARCH=arm64 ./scripts/build_binary.sh ./bin/linux-arm64
+	@echo "Built local-container-endpoints for linux-arm64"
 
 .PHONY: release
-release:
+release: release-amd release-arm
+
+.PHONY: release-amd
+release-amd:
+	docker run -v $(shell pwd):/usr/src/app/src/github.com/awslabs/amazon-ecs-local-container-endpoints \
+		--workdir=/usr/src/app/src/github.com/awslabs/amazon-ecs-local-container-endpoints \
+		--env GOPATH=/usr/src/app \
+		--env ECS_RELEASE=cleanbuild \
+		golang:$(GO_VERSION) make $(AMD_BINARY)
+	docker build -t amazon/amazon-ecs-local-container-endpoints:latest-amd64 .
+	docker tag amazon/amazon-ecs-local-container-endpoints:latest amazon/amazon-ecs-local-container-endpoints:$(TAG)-amd64
+	docker tag amazon/amazon-ecs-local-container-endpoints:latest amazon/amazon-ecs-local-container-endpoints:$(VERSION)-amd64
+
+.PHONY: release-arm
+release-arm:
 	docker run -v $(shell pwd):/usr/src/app/src/github.com/awslabs/amazon-ecs-local-container-endpoints \
 		--workdir=/usr/src/app/src/github.com/awslabs/amazon-ecs-local-container-endpoints \
 		--env GOPATH=/usr/src/app \
 		--env ECS_RELEASE=cleanbuild \
-		golang:1.12 make $(LINUX_BINARY)
-	docker build -t amazon/amazon-ecs-local-container-endpoints:latest .
-	docker tag amazon/amazon-ecs-local-container-endpoints:latest amazon/amazon-ecs-local-container-endpoints:$(TAG)
-	docker tag amazon/amazon-ecs-local-container-endpoints:latest amazon/amazon-ecs-local-container-endpoints:$(VERSION)
+		golang:$(GO_VERSION) make $(ARM_BINARY)
+	docker build -t amazon/amazon-ecs-local-container-endpoints:latest-arm64 .
```

Comment:
If I am following this right, we might need to also update:
https://github.com/awslabs/amazon-ecs-local-container-endpoints/blob/6a6717bf62abf7f173aa528d2ecedda94a3ea7f5/Dockerfile#L8

Right now it's going to use the x86 binary and not arm I think even when running `make release-arm`

Reply:
You're right -- I realized that we aren't actually passing in the GOARCH argument to the build_binary.sh, but I think there might be an easier way to do this using docker [buildx](https://docs.docker.com/buildx/working-with-buildx/) so gonna give that a shot.

Reply:
I think `buildx` is cool but would recommend against relying on it for anything (yet?) as it's still "experimental"